core/natives/src/device/init.rs

```diff
@@ -374,6 +380,15 @@ fn configure_device(device: &mut DeviceConfigurator) -> Result<Option<DeviceConf
     }
     device.add_extension(&push_descriptor_name);
 
+    let request_portability = cfg!(target_os="macos");
+    if request_portability {
+        let portability_subset_name = CString::new("VK_KHR_portability_subset").unwrap();
+        if !device.is_extension_supported(&portability_subset_name) {
```

Comment:
If a device doesn't have portability subset that doesn't mean that it won't work but that it doesn't need portability subset. So instead of failing if the extension is missing you should instead only query the extension if it is present.